kubejs/server_scripts/base/mods/craftingOnAStick.js

```diff
@@ -0,0 +1,17 @@
+if (feature('Remove crafting tables on a stick')) {
+    removeRecipe({mod:'crafting_on_a_stick'})
+    let namespace = (string) => {
```

Comment:
I dont like ommiting the modname as it makes it harder to refactor later. Also in this case it doesnt save you work or adds redability so i dont think it helps you anyway. I'd prefer we use the full ids and forgo this function